unit_tests/testIOutil/testIOutil.f90

```diff
@@ -551,21 +551,21 @@ SUBROUTINE testIO_Strings()
       ASSERT_EQ(str(-5000.0_SDK,5),'-5.00000E+03','str(SDK,nDecimal)')
 
       COMPONENT_TEST('getFloatFormat')
-      ASSERT_EQ(getFloatFormat(0.0_SRK),0_SIK,'zero')
-      ASSERT_EQ(getFloatFormat(-0.0_SRK),0_SIK,'zero')
-      ASSERT_EQ(getFloatFormat(0.1_SRK),0_SIK,'zero')
-      ASSERT_EQ(getFloatFormat(-0.1_SRK),1_SIK,'zero')
+      ASSERT_EQ(getFloatFormat(0.0_SRK),1_SIK,'zero')
+      ASSERT_EQ(getFloatFormat(-0.0_SRK),1_SIK,'zero')
```

Comment:
Hmm, this seems like it could be a problem.  I'm not sure this value would print properly if you told it to use 1 digit left of the decimal.  But I'm not sure the best way of handling it since `-0.0` does show up from time to time...